### PR TITLE
Completed es6-classes

### PR DESCRIPTION
**Pull request by K-Bowler**
image
image
image

### CODE REVIEW

File: es6-classes/example-class.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// }
+
+// const example = new ExampleClass();
+
+class ExampleClass {
+  constructor(example) {
+    this.example = example;
+    return example;
+  }
+}
```
Comment (scbowler) [inline es6-classes/example-class.js]: The `ExampleClass` should be an empty class:

```JAVASCRIPT
class ExampleClass {

};
```

File: es6-classes/stopwatch.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  seconds -= (secondsPerMinute * minutes);
+  hours = hours.toString().padStart(2, '0');
+  minutes = minutes.toString().padStart(2, '0');
+  seconds = seconds.toString().padStart(2, '0');
+  return `${hours}:${minutes}:${seconds}`;
+};
+
+Stopwatch.prototype.reset = function () {
+  this.elapsedSeconds = 0;
+};
```
Comment (scbowler) [inline es6-classes/stopwatch.js]: With classes we don't need to prototype the methods on, they get defined inside the class.

```JAVASCRIPT
class SomeClass {
  constructor() {

  }

  aMethod() {

  }

  bMethod() {

  }

  cMethod() {

  }
}
```

File: es6-classes/student.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Student.prototype.getFullName = function () {
+  const { firstName, lastName } = this;
+  return `${firstName} ${lastName}`;
+};
+
+Student.prototype.getIntroduction = function () {
+  const { subject } = this;
+  const fullName = this.getFullName();
+  return `Hello, my name is ${fullName} and I am studying ${subject}.`;
+};
```
Comment (scbowler) [inline es6-classes/student.js]: Same here, the methods should be defined within the class

**Review by scbowler — CHANGES_REQUESTED** [review]
A few updates needed, let me know if you have any questions.

**Review by scbowler — CHANGES_REQUESTED** [review]
`Student` class looks good, but looks like you missed the changes for the `Stopwatch` and `ExampleClass` classes.

**Comment by K-Bowler** [comment]
image

**Review by scbowler — APPROVED** [review]
Great job implementing ALL of the feedback, looks good! 👍

**Pull request merged**